client/devdocs/design/wordpress-components-gallery/angle-picker-control.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { AnglePickerControl } from '@wordpress/components';
+
+const Example = () => {
+	const [ angle, setAngle ] = useState();
+	return <AnglePickerControl value={ angle } onChange={ setAngle } />;
+};
+
+export default Example;
```

Comment:
I've noticed all component examples are named `Example`. Should we have a different name for each component, like component examples in Calypso do? So we'd have `AnglePickerControlExample` here, for example.

Reply:
I'll rename the default exports for each one to match the file name.